cmd/kubeadm/app/cmd/token.go

```diff
@@ -381,3 +387,16 @@ func getClientset(file string, dryRun bool) (clientset.Interface, error) {
 	}
 	return kubeconfigutil.ClientSetFromFile(file)
 }
+
+func findExistingKubeConfig(file string) string {
+	// The user did provide a --kubeconfig flag. Respect that and threat it as an
+	// explicit path without building a DefaultClientConfigLoadingRules object.
+	if file != defaultKubeConfig {
```

Comment:
@neolit123 can you move this function to `cmd/kubeadm/app/cmd/util` please?
We have other places in kubeadm where it's used as a Kubernetes client tool, and we could/should leverage the same functionality to be consistent